openapi.yaml

```diff
@@ -5611,6 +5624,47 @@ components:
       example:
         - href: 'https://example.openeo.org/docs/errors/SampleError'
           rel: about
+    log_level:
+      description: |-
+        The severity level of the log entry.
+
+        The order of the levels is as follows (from low to high severity): `debug`, `info`, `warning`, `error`.
+        
+        The level `error` usually stops processing the data.
+      type: string
+      enum:
+        - error
+        - warning
+        - info
+        - debug
+      example: error
+    min_log_level_default:
+      description: |-
+        The minimum severity level for log entries that the back-end stores for the processing request.
+
+        The order of the levels is as follows (from low to high severity): `debug`, `info`, `warning`, `error`.
+
+        That means if `warning` is set, the back-end will only store log entries with the level `warning` and `error`.
+      type: string
+      enum:
+        - error
+        - warning
+        - info
+        - debug
+      default: debug
```

Comment:
Using "debug" level by default seems wasteful and bad for the default signal to noise ratio of the logs (especially because all logs are returned by default). I would at least use "info" by default.

Reply:
Yes, I was also thinking about this but I had two reasons for including debug (for now):
1. It was the behavior before this addition and it would not be ideal to change the behavior in-between minor versions.
2. The `debug` process is having the `debug` level assigned. Requiring a user to specifically enable the debug level just to see what he requested to log explicitly is confusing. (I'd be open to rename the `debug` process to `log` and assign it the `info` level though.)

Reply:
That's better, 
but still a bit strange that the default does not correspond with the recommendation.

Using "warning" by default would correspond to what I expect from a logging system, but maybe that will make user support harder.

Bit torn on this one

Reply:
Yeah, I understand that, but also don't have a good solution. Maybe we can just try it out for now and evolve from there in 2.0. 🤷‍♀️